src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -333,6 +337,11 @@ public bool VerifyData(Organization organization, IGlobalSettings globalSettings
                 organization.SmServiceAccounts == SmServiceAccounts;
             }
 
+            if (valid && Version >= 14)
+            {
+                valid = organization.LimitCollectionCreationDeletion == LimitCollectionCreationDeletion;
+            }
```

Comment:
Can you check with product on this one: should the user be able to change this setting on the self-hosted instance as well?

At the moment they can, however that could put it out of sync with the license file, and this validation would fail.

It also means that the setting value could bounce around unexpectedly if self-hosted and cloud have different values and they update their license file. (i.e. cloud will overwrite the self-hosted value.)

I think these should be editable in 1 location only, or they shouldn't be synced. But largely a product/UX question I think.

Reply:
@eliykat Updating this PR to include endpoint protection for cloud-only access.

Reply:
Thanks for following this up!